src/main/java/org/datascience/project/PlaylistMapper.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Author: Abuchi Okeke
+ * Date: 04/28/2020
+ * Mapper class for mapReduce algorithm to find matching 
+ * songs using items from user inputs
+ */
+
+package org.datascience.project;
+
+import java.io.IOException;
+import java.util.regex.Pattern;
+import org.apache.hadoop.mapreduce.Job;
+import org.apache.hadoop.mapreduce.Mapper;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.io.LongWritable;
+import org.apache.hadoop.io.Text;
+
+//import com.datascience.project.FindPlaylistsJob;  
```

Comment:
don't worry about imports. i can clean that up if it causes problems when I try to run the tests this afternoon